docs/getting-started/docker-playgrounds/flink_cluster_playground.md

```diff
@@ -23,5 +23,658 @@ specific language governing permissions and limitations
+[windows]({{ site.baseurl }}/dev/stream/operators/windows.html). The results are written to the 
+*output* topic. 
+
+There are six different `page`s and the **events are generated so that each window contains exactly 
+one thousand records**.
+
+{% top %}
+
+## Setup
+
+{% if site.version contains "SNAPSHOT" %}
+<p style="border-radius: 5px; padding: 5px" class="bg-danger">
+  <b>Note</b>: The Apache Flink Docker images used for this playground are only available for
+  released versions of Apache Flink. Since you are currently looking at the latest SNAPSHOT
+  version of the documentation the branch referenced below will not exist. You can either change it 
+  manually or switch to the released version of the ocumentation via the release picker.
+</p>
+{% endif %}
+
+In this section you will setup the playground locally on your machine and verify that the Job is 
+running successfully.
+
+This guide assumes that you have [docker](https://docs.docker.com/) (1.12+) and
+[docker-compose](https://docs.docker.com/compose/) (2.1+) installed on your machine.
+
+The required configuration files are available in the 
+[flink-playgrounds](https://github.com/apache/flink-playgrounds) repository. Check it out and spin
+up the environment:
+
+{% highlight bash %}
+git clone --branch release-{{ site.version }} git@github.com:apache/flink-playgrounds.git
+cd flink-cluster-playground
+docker-compose up -d
+{% endhighlight %}
+
+Afterwards, `docker-compose ps` should give you the following output:
+
+{% highlight bash %}
+                     Name                                    Command               State                   Ports                
+--------------------------------------------------------------------------------------------------------------------------------
+flink-cluster-playground_clickevent-generator_1   /docker-entrypoint.sh java ...   Up       6123/tcp, 8081/tcp                  
+flink-cluster-playground_client_1                 /docker-entrypoint.sh flin ...   Exit 0                                       
+flink-cluster-playground_jobmanager_1             /docker-entrypoint.sh jobm ...   Up       6123/tcp, 0.0.0.0:8081->8081/tcp    
+flink-cluster-playground_kafka_1                  start-kafka.sh                   Up       0.0.0.0:9094->9094/tcp              
+flink-cluster-playground_taskmanager_1            /docker-entrypoint.sh task ...   Up       6123/tcp, 8081/tcp                  
+flink-cluster-playground_zookeeper_1              /bin/sh -c /usr/sbin/sshd  ...   Up       2181/tcp, 22/tcp, 2888/tcp, 3888/tcp
+{% endhighlight %}
+
+This indicates that the client container has successfully submitted the Flink Job ("Exit 0") and all 
+cluster components as well as the data generator are running ("Up").
+
+## Interaction with the Playground
+
+There are many ways to interact with this playground and its components.
+
+### Flink WebUI
+
+The Flink WebUI is probably the most natural starting point to observe your Flink Cluster. It is
```

Comment:
I find the wording here strange. How about: 

> The most natural starting point to observe your Flink Cluster is the Web UI exposed under http://localhost:8081.

Reply:
Done.